hack/krmFunctionBenchmark/benchmark.sh

```diff
@@ -0,0 +1,67 @@
+#! /bin/bash
+
```

Comment:
Even though the markdown refers to this, please add a standalone
usage message if run with no args, e.g.

```
if [ "$1" != "--doIt" ]; then
  echo "Usage: $0 --doIt"
  echo " "
  echo "This script measures performance of ..."
  echo "It does so by ..."
  exit 1
fi
```

Alternatively, change the script to demand useful arguments (so we don't need the --doIt flag), e.g.
```
Usage: benchmark 10 100 1000
This script measures performance of ...
It does so by ...
```